frontend/views/DetailsPanel.js

```diff
@@ -36,89 +48,92 @@ module.exports = kind({
-              disabled: true,
-            },
-            {
-              kind: LabeledTextItem,
-              label: 'Root required',
-              name: 'rootRequired',
-              text: 'unknown',
-              disabled: true,
-            },
+          classes: 'moon-hspacing top', components: [
             {
               components: [
+                {kind: Divider, content: 'App information'},
                 {
-                  name: 'installButton', kind: ProgressButton, content: 'Install', progress: 0,
-                  style: 'width:100%;min-width:100%;', ontap: 'installApp',
+                  kind: LabeledTextItem,
+                  label: 'Version',
+                  name: 'version',
+                  text: 'unknown',
                   disabled: true,
                 },
-              ],
-            },
-            {
-              components: [
                 {
-                  name: 'launchButton', kind: ProgressButton, content: 'Launch', progress: 0,
-                  style: 'width:100%;min-width:100%;', ontap: 'launchApp',
+                  kind: LabeledTextItem,
+                  label: 'Root required',
+                  name: 'rootRequired',
+                  text: 'unknown',
                   disabled: true,
                 },
+                {
+                  components: [
+                    {
+                      name: 'installButton', kind: ProgressButton, content: 'Install', progress: 0,
+                      style: 'width:100%;min-width:100%;', ontap: 'installApp',
+                      disabled: true,
+                    },
+                  ],
+                },
+                {
+                  components: [
+                    {
+                      name: 'launchButton', kind: ProgressButton, content: 'Launch', progress: 0,
+                      style: 'width:100%;min-width:100%;', ontap: 'launchApp',
+                      disabled: true,
+                    },
+                  ],
+                },
+                {
+                  // WIP
+                  showing: false,
+                  components: [
+                    {
+                      name: 'uninstallButton', kind: Button, content: 'Uninstall', style: 'width:100%;min-width:100%', small: true,
+                      disabled: true,
+                      ontap: 'uninstallApp',
+                    },
+                  ],
+                },
               ],
+              classes: 'moon-6h',
             },
             {
-              // WIP
-              showing: false,
               components: [
+                {kind: Divider, content: 'Description'},
                 {
-                  name: 'uninstallButton', kind: Button, content: 'Uninstall', style: 'width:100%;min-width:100%', small: true,
+                  name: 'appDescription',
+                  kind: BodyText,
+                  content: 'No description provided for this package',
+                  allowHtml: true
```

Comment:
Can we specify css class here? I tried but didn't succeed. What I want to achieve is to make sure this element has `max-width` of 100%, including its images.

Reply:
`max-width: 100%` + `overflow: hidden`, yeah. That should be possible. `style` attribute should theoretically work in that case.

Reply:
Fixed, limited width to 100%, added overflow: hidden, limited width of all inner images to 100%.